From 7a27ce82a0199a9205471d380d48467adfdea731 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 24 Jul 2014 14:47:14 +0200
Subject: [PATCH] dracut.spec: fixed license file packaging

(cherry picked from commit caa72f829007a868bf47a9bb82a3e359acd6a420)
(cherry picked from commit 1ddd3fd89a4dea2a247dc824ad4da09d2493a74d)
---
 dracut.spec | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index cb27f37..8912938 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -30,6 +30,8 @@ URL: https://dracut.wiki.kernel.org/
 # Source can be generated by
 # http://git.kernel.org/?p=boot/dracut/dracut.git;a=snapshot;h=%{version};sf=tgz
 Source0: http://www.kernel.org/pub/linux/utils/boot/dracut/dracut-%{version}.tar.xz
+Source1: https://www.gnu.org/licenses/lgpl-2.1.txt
+
 
 BuildRequires: bash git
 
@@ -194,6 +196,7 @@ This package contains tools to assemble the local initrd and host configuration.
 
 %prep
 %setup -q -n %{name}-%{version}
+cp %{SOURCE1} .
 
 %if %{defined PATCH1}
 git init
@@ -293,8 +296,10 @@ rm -rf -- $RPM_BUILD_ROOT
 %files
 %defattr(-,root,root,0755)
 %if %{with doc}
-%doc README HACKING TODO COPYING AUTHORS NEWS dracut.html dracut.png dracut.svg
+%doc README HACKING TODO AUTHORS NEWS dracut.html dracut.png dracut.svg
 %endif
+%{!?_licensedir:%global license %%doc}
+%license COPYING lgpl-2.1.txt
 %{_bindir}/dracut
 # compat symlink
 %{_sbindir}/dracut
@@ -441,11 +446,6 @@ rm -rf -- $RPM_BUILD_ROOT
 
 %files fips-aesni
 %defattr(-,root,root,0755)
-
-%if %{with doc}
-%doc COPYING
-%endif
-
 %{dracutlibdir}/modules.d/02fips-aesni
 
 %files caps
